osi_common.proto

```diff
@@ -428,3 +428,210 @@ message BaseMoving
     //
     repeated Vector2d base_polygon = 7;
 }
+
+//
+// \brief A description for the steeringwheel.
+//
+message Steeringwheel
+{
+    // Angle of the steeringwheel. 
+    //
+    // Unit in rad: 0=Central (Straight); Left>0; 0>Right.
```

Comment:
@CarloVanDriestenBMW: Right should be clockwise, if you take a look to the "Einheitskreis" or am I wrong here?

Reply:
The ISO 7401 describes tests in degrees. SI would be rad.
==> Open Point